data/en/census_household.json

```diff
@@ -1833,22 +1833,22 @@
                                 "options": [{
                                         "label": "White",
                                         "value": "White",
-                                        "description": "Including English / Welsh / Scottish / Northern Irish / British, Irish, Gypsy or Irish Traveller or any other White background"
+                                        "description": "Including English, Welsh, Scottish, Northern Irish, British, Irish, Gypsy, Irish Traveller or any other White background"
                                     },
                                     {
-                                        "label": "Mixed / multiple ethnic groups",
-                                        "value": "Mixed / multiple ethnic groups",
-                                        "description": "Including White and Black Caribbean, White and Black African,  White and Asian or any other Mixed/multiple ethnic background"
+                                        "label": "Mixed or multiple ethnic groups",
+                                        "value": "Mixed or multiple ethnic groups",
+                                        "description": "Including White and Black Caribbean, White and Black African, White and Asian or any other Mixed/multiple ethnic background"
```

Comment:
shouldn't this be `Mixed or multiple`?

Reply:
Thats a content thing all the card said was to replace / with or etc.

Reply:
@MIslam92 it is a `/` ? so should be replaced? you have on other descriptions in this PR